### PR TITLE
package: add urls for npm

### PR DESCRIPTION
**Pull request by dcousens**

### CODE REVIEW

**Comment by dcousens** [comment]
Please merge or close.

File: package.json [hunk 1]
```diff
@@ -6,6 +6,14 @@
...
     "test": "make test-unit"
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/scivey/async-limit.git"
+  },
+  "bugs": {
+    "url": "https://github.com/scivey/async-limit/issues"
+  },
+  "homepage": "https://github.com/scivey/async-limit"
```
Comment (dcousens) [inline package.json]: Missing comma here.  I don't have the fork any more, but should be an easy addition.

**Comment by dcousens** [comment]
Ping ping

**Comment by dcousens** [comment]
Ping ping

**Comment by dcousens** [comment]
Ping ping @scivey

**Comment by dcousens** [comment]
Ping ping ping @scivey

**Comment by scivey** [comment]
image

**Comment by dcousens** [comment]
@scivey may I ask why?
Is there anything you want changed?

**Pull request closed without merging**